### PR TITLE
Update the Search Synonyms topic with possible action information

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) updates the Search Synonyms topic with possible action information

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/marketing/search-terms.html

### CODE REVIEW

File: src/marketing/search-synonyms.md [hunk 1]
```diff
@@ -39,3 +39,31 @@ _Search Results_
...
+
+1. When complete, click <span class="btn">Submit</span>.
+
+**Method 2:** On the Edit a Synonym Group page:
+
+1. In the Search Synonyms grid, click the row of any record to open the synonym group in edit mode.
+
+1. Click <span class="btn">Delete Synonym Group</span>.
+
+1. Confirm the deletion in the pop-up window.
```
Comment (leslietilling) [inline src/marketing/search-synonyms.md:69]:
```suggestion
## Edit a synonym group

1. In the _Search Synonyms_ grid, click the row of any record to open the synonym group in edit mode.

1. Make the necessary changes.

1. When complete, click <span class="btn">Save Synonym Group</span>.

## Delete a synonym group

There are two methods for deleting a synonym group---from the grid and on the edit page.

**Method 1:** In the Search Synonyms grid

1. In the _Search Synonyms_ grid, select the checkbox of the group to be deleted.

1. In the upper-left corner of the list, set **Actions** to `Delete`.

1. When complete, click <span class="btn">Submit</span>.

**Method 2:** On the Edit a Synonym Group page

1. In the Search Synonyms grid, click the row of any record to open the synonym group in edit mode.

1. Click <span class="btn">Delete Synonym Group</span>.

1. When prompted, confirm the removal of the group.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edits for style and consistency

File: src/marketing/search-synonyms.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/marketing/search-synonyms.md]: Thank you @leslietilling , I have implemented your edits, please approve this PR 😊
Thank you!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**